index.js

```diff
@@ -1,18 +1,62 @@
+
 const readline = require('readline');
 const rl = readline.createInterface(process.stdin, process.stdout);
 
+//FUNCTIONS..........................................................
 function ask(questionText) {
   return new Promise((resolve, reject) => {
     rl.question(questionText, resolve);
   });
 }
+//...................................................................
+function randomInteger(min, max) {
+  let range = max - min + 1;
+  return min + Math.floor(Math.random() * range);
+}
+//...................................................................
+let max =  100
+//...................................................................
+let min = 1
+//...................................................................
+let guessCount = 1
 
-start();
 
+start();
 async function start() {
-  console.log("Let's play a game where you (human) make up a number and I (computer) try to guess it.")
-  let secretNumber = await ask("What is your secret number?\nI won't peek, I promise...\n");
-  console.log('You entered: ' + secretNumber);
-  // Now try and complete the program.
-  process.exit();
+  console.log("\nLet's play a game where you (human) make up a number and I (computer) try to guess it.")
+  let start = await ask('\n"Press" Enter To Start, Mwahaha!');
+  let secretNumber = await ask('\nWhat is your secret number?' + "\n\nI won't peek, I promise..." + '\n\nEnter a number between 1-100 and "Press" Enter.\n\n');
+  console.log('\nYou entered: ' + secretNumber);
+  let guess = randomInteger(min, max)
+  let firstGuess = await ask('\nIs it...' + guess + '?' + " Yes, or No: ")
+
+  
+  if (firstGuess === "N" || firstGuess === "No" || firstGuess === "NO" || firstGuess === "no" || firstGuess === "n") {
```

Comment:
Good use of the 'or' operator to check for different user inputs. Another way to do this would be to use an array and the includes method.
```js
let noResponses = ['N', 'No', 'NO', 'no', 'n']
if ( noResponses.includes( firstGuess ) ) {
  // do stuff
}
```